packages/devextreme/js/__internal/grids/grid_core/editing/m_editing.ts

```diff
@@ -159,16 +160,20 @@ class EditingControllerImpl extends modules.ViewController {
     this.component._optionsByReference[EDITING_CHANGES_OPTION_NAME] = true;
   }
 
-  getEditMode() {
-    const editMode: any = this.option('editing.mode');
-
+  getEditMode(): string {
+    const editMode = this.option('editing.mode') ?? EDIT_MODE_ROW;
     if (EDIT_MODES.includes(editMode)) {
       return editMode;
     }
 
     return EDIT_MODE_ROW;
   }
 
+  isCellBasedEditMode(): boolean {
+    const editMode: string = this.getEditMode();
```

Comment:
```suggestion
    const editMode: GridsEditMode = this.getEditMode();
```

or even better 

```suggestion
    const editMode = this.getEditMode();
```

since type would be inferred